### DISCUSSION TITLE
Total Yield Day drops to 0 when inverter is unreached at night

### DISCUSSION THREAD

**Discussion by sfritz67** [tbnobody/OpenDTU#1486] [Q&A]
Hello,
Everyday, my Total Yield Day drops to 0 when Total Power drops to 0 and inverter is not reached anymore, but I would like it to stay until end of the day.
is there a way to hold Total Yield Day count until midnight ?

Thanks,

sfritz67

**Comment by tbnobody** [comment]
Please see https://github.com/tbnobody/OpenDTU/wiki/FAQ#q-why-get-the-daily-yield-values-sometimes-reset-to-zero-at-the-evening

**Comment by stefan123t** [comment]
@tbnobody wouldn’t it be possible to calculate the current daily yield using the difference between the current_total_yield - last_start/midnight total_yield ?
We could even correct the last_start_total_yield or midnight_total_yield during/after start of the inverter if the initial daily_yield values reported by the MI are already > the above difference.
